sound/soc/intel/boards/sof_da7219_max98373.c

```diff
@@ -72,11 +72,17 @@ static const struct snd_kcontrol_new controls[] = {
 static const struct snd_kcontrol_new m98360a_controls[] = {
```

Comment:
@yongzhi1 Thanks. The patch now looks good and seems we have a consensus this is the right approach. I was about to merge this, but then I realized the commit message now seems a bit off. Could you please a short sentence to explain why the patch moves things around (i.e. "Spk Switch" is only valid for m98360a, so add separate widget tables for max98360a and max98373 -- or something to that effect). Thanks!

Reply:
+1 on the commit subject, I overlooked it. Thanks!

Reply:
Thanks all for your review, please let me know if above commit message works.